### DISCUSSION TITLE
12 hour time

### DISCUSSION THREAD

**Discussion by DRM001** [maruohon/minihud#189] [Ideas]
Hi,
It'd be great if you could add 12 hour time with AM/PM for both IRL and in-game time as an option.

Thanks

**Comment by maruohon** [comment]
You can set the IRL time to any format you want, using the format codes the Java SimpleDateFormat supports.
See: https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

So set Generic -> `dateFormatReal` to `yyyy-MM-dd KK:mm:ss a` to get the date and time in the AM/PM format.

For the in-game time, I guess I would need to add new value placeholders.
